openpype/pipeline/farm/pyblish_functions.py

```diff
@@ -579,12 +579,16 @@ def _create_instances_for_aov(instance, skeleton, aov_filter, additional_data,
 
         # if there are multiple cameras, we need to add camera name
         if isinstance(col, (list, tuple)):
-            cam = [c for c in cameras if c in col[0]]
+            cam = next((c for c in cameras if c in col[0]), None)
         else:
             # in case of single frame
-            cam = [c for c in cameras if c in col]
+            cam = next((cam for cam in cameras if cam in col), None)
         if cam:
             if aov:
+                # if there is duplicatd camera name found in aov,
+                # it would be removed
+                if aov.startswith(cam):
+                    aov = aov.replace(f"{cam}_", "")
```

Comment:
> Hmm - wait I see what you're doing. but how come `aov` already contains the camera name? That's wrong! :)
Maybe I can try to set up the camera folder to store the information. 
But not sure if it will work.

Reply:
I have tried to set the things up but more than one collections would be found by this case if I included multiple camera. This will error out the submit publish job. I tried to edit to have more than one collections, and then it messed up the naming and it will give you wrong metadata.json.
I am not sure if the hosts did the multiple camera rendering in different manner, 3dsmax needs to have batch render layer set up for multiple cameras rendering for manual setup. This PR actually sets up the multiple cameras in batch render dialog and uses the filepath as the render output. One other thing is, you can't put all the filepaths with multiple cameras altogether as it will create more than one collection when it pass to the clique.assemble.
So what I did is to separate beauty or other aovs by cameras to make sure just one collection happened and filepath with different cameras are really stored in the expectedFiles. 
Maybe there is some better way to manage the multiple camera renderings. 
Any thoughts? @antirotor @BigRoy

Reply:
I have done some tests with expectedFiles where there are two cameras involved. There are always more than one collections. And it seems that the cameras isn't iterated as if the aov does as aov is the key of the expectedFiles dict while expectedFiles is the value of the dict. Metadata json is just with the expectedFiles of one camera. From my current commit using the combination of aov and cameras as key of the dict. The commit i did is always one collection. But it will create duplciated camera name, so need some renames.